source/amazon/services/troubleshooting.rst

```diff
@@ -8,46 +8,232 @@ Troubleshooting
         :class: output
 
-        2019/10/28 13:58:10 wazuh-modulesd:aws-s3[8184] wm_aws.c:48 at wm_aws_main(): INFO: Module AWS started
+        2022/03/04 00:00:00 wazuh-modulesd:aws-s3: INFO: Module AWS started
+        2022/03/04 00:00:00 wazuh-modulesd:aws-s3: INFO: Starting fetching of logs.
 
 
-#. Scheduled scan:
+- When Scheduled scan is set:
 
     .. code-block:: none
         :class: output
 
-        2019/10/28 13:58:10 wazuh-modulesd:aws-s3: INFO: Starting fetching of logs.
-        2019/10/28 13:38:11 wazuh-modulesd:aws-s3: INFO: Fetching logs finished.
+        2022/03/04 00:00:00 wazuh-modulesd:aws-s3: INFO: Starting fetching of logs.
+        2022/03/04 00:00:00 wazuh-modulesd:aws-s3: INFO: Fetching logs finished.
 
 
-Common errors
--------------
+- When the module has finished its execution and is waiting until the :ref:`aws_interval` condition is met:
 
-#. Errors in ``ossec.log``
+    .. code-block:: none
+        :class: output
+
+        2022/03/04 00:00:00 wazuh-modulesd:aws-s3: INFO: Fetching logs finished.
+
+.. _aws_debug_mode:
+
+Enabling debug mode
+-------------------
+
+It is possible to obtain additional information about the module's execution by enabling the debug mode. This is used to see ``INFO``or ``DEBUG`` messages. There are three different debug levels available:
+
+- **Debug level 0**: Only ``ERROR`` and ``WARNING`` messages are written in the ``ossec.log`` file. This is the default value.
+
+- **Debug level 1**: In addition to ``ERROR`` and ``WARNING`` messages, ``INFO`` messages are written in the ``ossec.log`` file too. They are useful to check the execution of the module without having to manage large amounts of ``DEBUG`` messages.
+
+- **Debug level 2**: This is the highest level of verbosity. Every message type is dump into the ``ossec.log`` file, including ``DEBUG`` messages which contain the details of the different operations performed by the module. This is the recommended mode when troubleshooting the module.
+
+
+Follow these steps to enable debug mode:
+
+#. Add the following line to the ``WAZUH_PATH/etc/local_internal_options.conf`` file, specifying the desired debug level:
+
+    .. code-block:: none
+
+        wazuh_modules.debug=2
+
+
+#. Restart the Wazuh service
+
+
+.. tabs::
+
+  .. group-tab:: Wazuh manager
+
+      .. tabs::
+
+        .. tab:: For Systemd
+
+          .. code-block:: console
+
+            systemctl restart wazuh-manager
+
+        .. tab:: For SysV Init
+
+          .. code-block:: console
+
+            service wazuh-manager restart
+
+  .. group-tab:: Wazuh agent
+
+      .. tabs::
+
+        .. tab:: For Systemd
+
+          .. code-block:: console
+
+            systemctl restart wazuh-agent
+
+        .. tab:: For SysV Init
 
-    When an error occurs when trying to collect and parse logs for an AWS service, the ``ossec.log`` will output an error such as below:
+          .. code-block:: console
+
+            service wazuh-agent restart
```

Comment:
Consider adding this to `/source/_templates/common/restart_manager_or_agent.rst` as a template and including it here using
```
.. include:: ../../_templates/common/restart_manager_or_agent.rst
```

Reply:
Good idea. I have created the `/_templates/common/restart_manager_or_agent.rst` template for this, so it can be reused in the future.